src/pages/test-report/components/timer/timer.scss

```diff
@@ -24,7 +24,7 @@ timer {
     &.paused {
       color: map-get($colors-gds, 'gds-light-blue');
     }
-    text-align: center;
+    position: absolute;
```

Comment:
Is this to stop the text wobbling? Because a `1` character takes up less space than a `5` ?

Reply:
Yep, prevents the wobble/bounce